confite/__init__.py

```diff
@@ -59,12 +59,20 @@ def as_int(self, key) -> int or None:
     # -----------------------------------------------------
     # AS FILE
     # -----------------------------------------------------
-    def as_file(self, file_name: str, property_key: str, base64_decode=False):
+    def as_file(self, file_name: str, property_key: str, base64_decode=False, break_line_character='\\n'):
         with open(file_name, mode='w', encoding='utf-8') as property_file:
             if base64_decode:
                 config_property = self.as_base64_decoded_str(property_key)
             else:
                 config_property = self.config_map[property_key]
+
+            # Multiline Handler
+            lines_list: list[str] = config_property.split(break_line_character)
```

Comment:
This looks good, I would just extract this logic into an external function so that it is modular and we keep single responsibility on each function.

Reply:
done